uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/components/fields-templates/leave-type-wizard-field-allow_overuse.html

```diff
@@ -0,0 +1,8 @@
+<div class="col-md-12">
+  <div class="checkbox checkbox-switch">
+    <label>
+      <input type="checkbox" ng-model="field.value">
+      <span class="checkbox-switch-label"></span>
+    </label>
+  </div>
+</div>
```

Comment:
This field is a switch that, if switched on, shows other 2 fields, one of the dependent fields you referred to.

> Why can't we use generic fields instead of very specific ones with the same markup?

Please clarify, not sure I understand the question.